_test_/index.test.tsx

```diff
@@ -1,9 +1,20 @@
-﻿import React from 'react';
+﻿import React, {FunctionComponent} from 'react';
 import {render} from '@testing-library/react';
 import Home from "../pages";
+import { RouterContext } from 'next-server/dist/lib/router-context'
 
-test('Check that the home page renders', () => {
-    const home = render(<Home/>);
+
+test('Check that the home page renders', async () => {
+   const router = {
+       query:{token: "undefined"},
+   };
+    const home = render(
+        <RouterContext.Provider value={router.query}>
```

Comment:
I don't think this is quite the right value to be using - causes this error when the tests run.

![image](https://user-images.githubusercontent.com/8644502/80949179-689a6000-8deb-11ea-8409-1a09fb837f87.png)